changelog/README.md

```diff
@@ -3,6 +3,9 @@
 ## 7th October 2022
 
 * Added [Pagination](../guidelines/pagination.md) to [Get all resource access tokens](../operations/resourceaccesstokens.md#get-all-resource-access-tokens).
+* Added new page [Loyalty Programs](../operations/loyaltyprograms.md) under [Operations](../operations/README.md).
```

Comment:
As stated in previous comment, here also it should be:
* Added new page [Loyalty programs]